extensions/ql-vscode/src/databases/git-hub-url-identifier-helper.ts

```diff
@@ -0,0 +1,47 @@
+import { Uri } from "vscode";
```

Comment:
[As discussed on Slack—mentioning it here again for when you're back ❤️ ]
We could split the nwo/identifier changes into a separate PR ⚡ 

If you're doing that, maybe you could incorporate a previous suggestion about moving the file to `src/common/vscode`: https://github.com/github/vscode-codeql/pull/1865#pullrequestreview-1215686657?